templates/collection.nomad

```diff
@@ -236,6 +236,10 @@ job "collection-${name}" {
           sleep 5
           exit 1
         fi
+        date
+        ./manage.py migrate --noinput
+        ./manage.py healthcheck
```

Comment:
Since we're running it here, we should also remove it from the job: https://github.com/liquidinvestigations/node/blob/b91d76371b28f1a2176d386d39bbeaa4888b0fe2/templates/collection.nomad#L303